presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +168,10 @@ docker version
 
 stop_all_containers
 
+if [[ "$CONTINUOUS_INTEGRATION" == 'true' ]]; then
```

Comment:
We haven't added Docker and product tests to our Jenkins yet. Though I suppose it won't be hard to export some extra environment variables.